tests/sles4sap/crash/azure/crash_azure.pm

```diff
@@ -0,0 +1,69 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary:  Executes a crash scenario on azure.
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use mmapi 'get_current_job_id';
+use serial_terminal 'select_serial_terminal';
+use version_utils 'is_sle';
+use sles4sap::azure_cli;
+use publiccloud::instance;
+use utils;
+
+use constant DEPLOY_PREFIX => 'clne';
+
+sub run {
+    my ($self) = @_;
+
+    # Crash test
+    my $ssh_cmd = get_required_var('SSH_CMD');
+    my $vm_ip = get_required_var('VM_IP');
+    my $instance = publiccloud::instance->new(public_ip => $vm_ip);
+
+    record_info('PATCH', 'Fully patch system start');
+    my $remote = '-o ControlMaster=no -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ' . 'cloudadmin' . '@' . $vm_ip;
+    ssh_fully_patch_system($remote);
+    assert_script_run(join(' ',
+            $ssh_cmd,
+            'sudo',
+            'reboot'),
```

Comment:
Isn't there a reboot function already in a library? See: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/publiccloud/instance.pm#L561